packages/editor/src/components/editor-snackbars/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { filter } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { SnackbarList } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { store as noticesStore } from '@wordpress/notices';
+
+export default function EditorSnackbars() {
```

Comment:
Do you think it should be `EditorSnackbarNotices` (I'm not great with naming, just asking :) )

Reply:
> I'm not great with naming, just asking :)
Same here 😄 

Since "Snackbars" or "Toasts" are notice types themself, I don't think there's a need for the `Notices` suffix IMO.